ProcessMaker/Models/ProcessRequest.php

```diff
@@ -211,19 +211,29 @@ public function __construct(array $argument = [])
     }
 
     /**
-     * Validation rules.
+     * Get the validation rules for process requests.
      *
-     * @param null $existing
+     * @param mixed|null $existing ID of existing process request to ignore in unique validation
      *
-     * @return array
+     * @return array Array of validation rules for the process request fields:
+     *               - name: Required string, max 100 chars, alpha spaces only, unique unless updating existing
+     *               - data: Required field
+     *               - status: Must be one of: ACTIVE, COMPLETED, ERROR, CANCELED
+     *               - process_id: Required and must exist in processes table
+     *               - process_collaboration_id: Optional but must exist in process_collaborations table if provided
+     *               - user_id: Must exist in users table
      */
     public static function rules($existing = null)
     {
         $self = new self();
         $unique = Rule::unique($self->getConnectionName() . '.process_requests')->ignore($existing);
```

Comment:
This `unique` validation is not consistent with data in process_requests table, so please remove the unique validation for the name field.